plugin/system_copy.vim

```diff
@@ -2,6 +2,10 @@ let s:blockwise = 'blockwise visual'
 let s:visual = 'visual'
 let s:motion = 'motion'
 let s:linewise = 'linewise'
+let s:mac = 'mac'
+let s:windows = 'windows'
+let s:linux = 'linux'
+let s:os = ''
```

Comment:
We can get rid of this if you remove the caching in the os lookup function below.